### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by nahnahdev**
## Project requirements

- [ ] Watch the video with the UI example again (you have seen it while reading the Sneak Peek).

- [ ] Your console app should behave in the same way as it is in the example.

- [ ] Create an `app.rb` file to serve as your console app entry point. It should have methods that do the following:

> List all books.
> List all people.
> Create a person (teacher or student, not a plain Person).
> Create a book.
> Create a rental.
> List all rentals for a given person's id.

- [ ] In your `main.rb `define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:

> Present the user with a list of options to perform.
> Lets users choose an option.
> If needed, ask for parameters for the option.
> Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    end
+
+    puts 'Press any key to continue'
+    gets.chomp
+  end
+end
+
+def get_menu_option(options)
+  puts 'Please choose an option by entering a number:'
+  options.each_with_index { |(option, _), index| puts "#{index}) #{option}" }
```
Comment (sja-thedude) [inline app.rb]: - [ ] Kindly make sure to add a `main.rb` file which will work as the entry point as stated in the requirements.

Screenshot 2022-12-22 at 2 38 50 PM

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+          puts "[Student]: Name: #{person.name}, Id: #{person.id}, Age: #{person.age}"
+        else
+          puts "[Teacher]: Name: #{person.name}, Id: #{person.id}, Age: #{person.age}"
+        end
+      end
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
```
Comment (sja-thedude) [inline app.rb:48]: - [ ] Kindly make sure to fix the error as shown in the below video. When I click an option for example 2 it displays the entire list and when I click 2 again it displays the required function with it. Make sure that app works smoothly.

https://user-images.githubusercontent.com/70269507/209128275-82c171fe-060d-4763-8d19-b7a4031ba778.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @NahnahAJ, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Correct Gitflow
✔️ Professional Readme
✔️ All linters passing

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
Hi @NahnahAJ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**